README.md

```diff
@@ -1,13 +1,14 @@
 # Ppxlib - Meta-programming for OCaml
 
-[![Travis status][travis-img]][travis] [![AppVeyor status][appveyor-img]][appveyor]
+[![ocaml-ci status][ocaml-ci-img]][ocaml-ci] [![AppVeyor status][appveyor-img]][appveyor]
 
-[travis]:         https://travis-ci.org/ocaml-ppx/ppxlib
-[travis-img]:     https://travis-ci.org/ocaml-ppx/ppxlib.svg?branch=master
+[ocaml-ci]: https://ci.ocamllabs.io/github/ocamllabs/dune-release
```

Comment:
Did you maybe mean `https://ci.ocamllabs.io/github/ocaml-ppx/ppxlib`?

Reply:
Wooups!